README.md

```
@@ -1,38 +1,90 @@
+docker compose up -d postgres redis
+make -C backend install
+make -C frontend install
+make dev                  # backend :8000, frontend :3000
 ```
 
-## Project structure
+Then open `http://localhost:3000`.
+
+## Project layout
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)